app/controllers/admin/tutorials_controller.rb

```diff
@@ -1,14 +1,24 @@
 # frozen_string_literal: true
 
 class Admin::TutorialsController < Admin::BaseController
-  def edit
-    @tutorial = Tutorial.find(params[:id])
+  def new
+    @tutorial = Tutorial.new
   end
 
-  def create; end
+  def create
+    new_tutorial = Tutorial.new(new_tutorial_params)
+    new_tutorial.save
+    if new_tutorial.save
+      flash[:success] = 'Successfully added tutorial.'
+      redirect_to tutorial_path(new_tutorial.id)
+    else
+      flash[:error] = 'Tutorial was not added.'
+      redirect_to new_admin_tutorial_path
+    end
```

Comment:
I am not sure why `render :new` isn't working for you here. I think that would be a better option of it works.